From e9a37b4ab7c3dee44a666b528075ba307a34c9d1 Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Mon, 9 Dec 2024 13:37:11 +0100
Subject: [PATCH 1415/1695] drivers: stm32_stgen: remove "st,stm32mp1-stgen"
 from the match table

Remove the "st,stm32mp1-stgen" compatible from the match table as the
current driver implementation is not functional for stm32mp1x platforms.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 665daa7fd245b1fb1891f17074da2a83fc5c9b32 drivers: stm32_stgen: remove "st,stm32mp1-stgen" from the match table]
Change-Id: Ib34d846005bd5cdc53ba4e10105108f8dd3bdd8c
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/423597
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 core/drivers/counter/stm32_stgen.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/core/drivers/counter/stm32_stgen.c b/core/drivers/counter/stm32_stgen.c
index da63cb679..938944d6b 100644
--- a/core/drivers/counter/stm32_stgen.c
+++ b/core/drivers/counter/stm32_stgen.c
@@ -251,9 +251,6 @@ static TEE_Result stgen_probe(const void *fdt, int node,
 }
 
 static const struct dt_device_match stm32_stgen_match_table[] = {
-	{
-		.compatible = "st,stm32mp1-stgen"
-	},
 	{
 		.compatible = "st,stm32mp25-stgen"
 	},
-- 
2.47.2

